src/dms-preview/azext_dms/custom.py

```diff
@@ -59,27 +65,21 @@ def create_or_update_project(
         # If not an extension scenario, run CLI core method
         # TODO: We currently don't have any CLI core code to perform any validations
         # because of this we need to raise the error here.
-        try:
```

Comment:
@temandr I took this out as it was causing a pylint issue with immediately raising in the except block. I think this should be ok because an exception will get raised to the client in the same way and we're still performing our validations needed in the extension.